knexfile.js

```diff
@@ -47,5 +47,12 @@ module.exports = {
     seeds: {
       directory: __dirname + "/db/seeds/production"
     }
-  }
+  },
+
+  onUpdateTrigger: table => `
+    CREATE TRIGGER ${table}_updated_at
+    BEFORE UPDATE ON ${table}
+    FOR EACH ROW
+    EXECUTE PROCEDURE on_update_timestamp();
+  `
```

Comment:
limiter au cas ou c'est le nb de mesures qui change `WHEN (OLD. mesures_en_cours IS DISTINCT FROM NEW. mesures_en_cours)`

bonus : ajouter un test 


https://www.postgresql.org/docs/10/static/sql-createtrigger.html